modules/monitoring-examining-metrics-as-a-developer.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="examining-metrics-as-a-developer_{context}"]
+= Examining metrics as a developer
+
+You might enable user workload monitoring for an application or service in a project. In that case, a developer of that project can examine the exposed custom metrics using the Developer Perspective in the Web console.
+
+:FeatureName: Examining metrics using the Developer Perspective
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+.Procedure
+
+. Open the {product-title} Web console, switch to the Developer Perspective, and navigate to the *Advanced* -> *Metrics* page. Select the project where the user workload resides.
```

Comment:
This is similar to what described at https://github.com/openshift/openshift-docs/pull/18788/files#diff-7c513e88469df85781bf83a97b00688aR21. Any reason why we have this duplication?

Reply:
You are right, deleted the redundancy in modules/monitoring-examining-metrics-as-a-developer.adoc.